vcd/resource_vcd_external_network.go

```diff
@@ -0,0 +1,304 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/validation"
```

Comment:
this package is not in the vendor directory

Reply:
Should be solved when run `go mod vendor` with new tagged govcd

Reply:
Please make Travis succeed first.